pkg/networks/usernet/gvproxy.go

```diff
@@ -54,7 +54,7 @@ func StartGVisorNetstack(ctx context.Context, gVisorOpts *GVisorNetstackOpts) er
 			}
 		}
 	}
-	leases[gatewayIP] = "5a:94:ef:e4:0c:df"
+	leases[gatewayIP] = "5a:94:ef:e4:0c:dd"
```

Comment:
Could you explain this?

Reply:
Its just a code smell change to match gateway mac addr to the one defined here
https://github.com/lima-vm/lima/blob/d722c35452e21b95fd51ce2092a615ab6c93bdd7/pkg/networks/usernet/gvproxy.go#L70

Reply:
Can we use a const?